packages/lib/services/webhook.ts

```diff
@@ -24,21 +28,32 @@ export const getWebhook = async (id: string): Promise<TWebhook | null> => {
       id,
     },
   });
-  return webhook;
+  if (webhook) {
+    return {
+      ...webhook,
+      source: webhook.source as "user" | "zapier" | null | undefined,
```

Comment:
if we have the right prisma types we can also skip these checks again and simplify the method again